src/Symfony/Component/Messenger/Tests/EnvelopeTest.php

```diff
@@ -114,6 +118,40 @@ public function testWrapWithEnvelope()
         $this->assertCount(1, $envelope->all(DelayStamp::class));
         $this->assertCount(1, $envelope->all(ReceivedStamp::class));
     }
+
+    /**
+     * To be removed in 6.0.
+     *
+     * @group legacy
+     */
+    public function testWithAliases()
+    {
+        $envelope = new Envelope(new \stdClass(), [
+            $s1 = new LegacyDoctrineReceivedStamp(1),
+            $s2 = new LegacyRedisReceivedStamp(2),
+            $s3 = new LegacyDoctrineReceivedStamp(3),
+        ]);
+
+        self::assertSame([
+            DoctrineReceivedStamp::class => [$s1, $s3],
+            RedisReceivedStamp::class => [$s2],
+        ], $envelope->all());
```

Comment:
Unfortunately, I think we cannot do much for the case someone is using `Envelope::all()` with no type argument and still using the alias instead of the new fqcn to do their things.